Add Platform operator concept content (#155)
RedHatInsights/red-hat-ansible-automation-platform-documentation: downstream/assemblies/platform/assembly-operator-install-planning.adoc

:context: operator-install-planning

[role="_abstract"]
{PlatformName} is supported on both Red Hat Enterprise Linux 8 and Red Hat Openshift.

OpenShift operators help install and automate day-2 operations of complex, distributed software on {OCP}. The {OperatorPlatform} enables you to deploy and manage {PlatformNameShort} components on {OCP}.

> You can use this section to help plan {PlatformName} installation on your {OCP} environment. Before installing, review the individual scenarios to determine which meets your requirements.

Reviewer: ```suggestion
You can use this section to help plan {PlatformName} installation on your {OCP} environment. Before installing, review the supported installation scenarios to determine which meets your requirements.
```

== About {OperatorPlatform}

The {OperatorPlatform} provides cloud-native, push-button deployment of new {PlatformNameShort} instances in your OpenShift environment. 
The {OperatorPlatform} includes resource types to deploy and manage instances of {ControllerNameStart} and Private Automation hub. It also includes {ControllerName} job resources for defining and launching jobs inside your {ControllerName} deployments. 

> You can install the {OperatorPlatform} from the Red Hat Operators catalog in OperatorHub.

Reviewer: ```suggestion
```

Reviewer: (Moving the "action item" from line 23 down to the end)

> Deploying {PlatformNameShort} instances with a Kubernetes native operator offers advantages over launching instances from a playbook deployed on {OCP}. The {OperatorPlatform} supports upgrades and provides full lifecycle support for your {PlatformName} deployments. 

Reviewer: ```suggestion
Deploying {PlatformNameShort} instances with a Kubernetes native operator offers several advantages over launching instances from a playbook deployed on {OCP}, including upgrades and full lifecycle support for your {PlatformName} deployments. 

You can install the {OperatorPlatform} from the Red Hat Operators catalog in OperatorHub.
```

:context: operator-install-planning

[role="_abstract"]
{PlatformName} is supported on both Red Hat Enterprise Linux 8 and Red Hat Openshift.

OpenShift operators help install and automate day-2 operations of complex, distributed software on {OCP}. The {OperatorPlatform} enables you to deploy and manage {PlatformNameShort} components on {OCP}.

> You can use this section to help plan {PlatformName} installation on your {OCP} environment. Before installing, review the supported installation scenarios to determine which meets your requirements.

Author: ```suggestion
You can use this section to help plan your {PlatformName} installation on your {OCP} environment. Before installing, review the supported installation scenarios to determine which meets your requirements.
```
